### PR TITLE
[FIX] ldap User Name

### PR DESCRIPTION
**Pull request by Siddhanttimeline**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v153/MigrationUtil.java [hunk 1]
```diff
@@ -11,4 +16,27 @@ public static void updateEmailTemplates(CollectionDAO collectionDAO) {
...
+      int total = daoCollection.userDAO().listTotalCount();
+      int offset = 0;
+      int limit = 200;
+      while (offset < total) {
+        List<String> userEntities = daoCollection.userDAO().listAfterWithOffset(limit, offset);
+        for (String json : userEntities) {
+          User userEntity = JsonUtils.readValue(json, User.class);
+          String email = userEntity.getEmail();
+          String emailPrefix = email.substring(0, email.indexOf("@"));
+          userEntity.setFullyQualifiedName(emailPrefix);
```
Comment (mohityadav766) [inline openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v153/MigrationUtil.java:33]: here don't put email prefix directly, use as below

```userEntity.setFullyQualifiedName(EntityInterfaceUtil.quote(emailPrefix)); ```

also update the name here without 

``` userEntity.setFullyQualifiedName(emailPrefix); ```

Comment (mohityadav766) [reply openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v153/MigrationUtil.java:33]: please note name won't need quote thing
Also you might need to if we need to resync the nameHash for the user_entity table after this update

**Review by mohityadav766 — COMMENTED** [review]
@Siddhanttimeline we need to also take a look at LdapAuthenticator and work with emailPrefix and not userDn , it might still be there please check

**Comment by Siddhanttimeline** [comment]
> @Siddhanttimeline we need to also take a look at LdapAuthenticator and work with emailPrefix and not userDn , it might still be there please check

Noted, checking.

**Review by mohityadav766 — APPROVED** [review]

**Pull request merged**